public/scss/cart.scss

```diff
@@ -0,0 +1,201 @@
+.cart_Box {
+  position: relative;
+  max-width: 770px;
+  width: 100%;
+  margin: 0px auto;
+  background-color: #eee;
+  padding: 30px 0px 0;
+
+  h3 {
+    font-size: 21px;
+    border-top: #e6e6e6;
+    padding: 14px 25px;
+
+    background-color: #fff;
+    span {
+      font-size: 20px;
+      font-weight: 300;
+      margin-left: 5px;
+    }
+  }
+  ul,
+  li {
+    display: block;
+  }
+  .items {
```

Comment:
Why you not following BEM methodology to create CSS classes/selectors?

Reply:
resolved